libexec/bats-core/bats

```diff
@@ -163,6 +163,9 @@ while [[ "$#" -ne 0 ]]; do
     # suppress stdout (on success)
     parallel --env _ echo ::: 1 >/dev/null || exit 1
     ;;
+  --dont-parallelize-across-files)
```

Comment:
I'd prefer the `--no-parallelize-across-file` which is IMHO clearer, because suite is very ambiguous wording and I can only wonder what a parallel file is.

This was intended as internal debugging aid. If we see this as a valuable option, then we document it, right?